### PR TITLE
Feature: Fast author-color generation

### LINKED ISSUE #687: Distinct colors is extremely slow
For linux it takes about 30-40 sec to generate colors for the 10k authors

### PR DESCRIPTION
**Pull request by tjomson**
Resolves #687 
Instead of distinct colors, we now generate colors by using author-names as seeds.
This means that author colors are consistent across branches and even repos!
If you don't like the author-colors, a button has been added to the legend to reroll the colors by adding a random seed. This seed is saved to truckconfig

### CODE REVIEW

**Review by joglr — APPROVED** [review]
LGTM!

File: src/analyzer/model.ts [hunk 1]
```diff
@@ -36,10 +37,11 @@ export interface TruckConfig {
   unionedAuthors: string[][]
   hiddenFiles: string[]
   invalidateCache: boolean
+  colorSeed?: string
```
Comment (mircealungu) [inline src/analyzer/model.ts:40]: seeds are usually numbers. why is this a string?

Comment (joglr) [reply src/analyzer/model.ts:40]: It seems that the library `uniqolor` takes either a UUID or anything that can be converted to a string or number, so I think using a string is fine?

Comment (mircealungu) [reply src/analyzer/model.ts:40]: yes. I saw that after!

**Pull request merged**